config/step/session/local.yaml

```
@@ -1,3 +1,3 @@
-defaults:
-    - session_config
+_target_: otg.common.session.Session
+spark_uri: local[*]
 hail_home: .venv/lib/python3.10/site-packages/hail/backend/hail-all-spark.jar
```

This was wrong, local hail_home doesn't have a fixed location like in Dataproc.
We usually find it after running: `poetry run pip show hail | grep Location | awk -F' ' '{print $2 "/hail"}'`
I suggest that this is required when in local model
```suggestion
hail_home: ???
```

---

Thank you! This makes sense